src/edu/stanford/dlss/was/WasapiDownloaderSettings.java

```diff
@@ -62,15 +68,16 @@ public WasapiDownloaderSettings(String settingsFileLocation, String[] args) thro
     try {
       loadPropertiesFile(settingsFileLocation);
       parseArgsIntoSettings(args);
-
-      //TODO: validate settings state.  see https://github.com/sul-dlss/wasapi-downloader/issues/42
+      validateSettings();
     } catch (IOException e) {
       throw new SettingsLoadException("Error reading properties file: " + e.getMessage(), e);
     } catch (ParseException e) {
       throw new SettingsLoadException("Error parsing command line arguments: " + e.getMessage(), e);
     }
   }
 
+  protected WasapiDownloaderSettings() { } //empty constructor, for testing
```

Comment:
i think `// for testing`  is sufficient  (we can see it's an empty constructor.  ;-)